parity/db/rocksdb/mod.rs

```diff
@@ -86,16 +86,16 @@ pub fn restoration_db_handler(client_path: &Path, client_config: &ClientConfig)
 	})
 }
 
-/// Open a new main DB.
-pub fn open_db(
+/// Open a new light client DB.
+pub fn open_db_light(
 	client_path: &str,
 	cache_config: &CacheConfig,
 	compaction: &DatabaseCompactionProfile
 ) -> io::Result<Arc<dyn BlockChainDB>> {
 	let path = Path::new(client_path);
 
 	let db_config = DatabaseConfig {
-		memory_budget: Some(cache_config.blockchain() as usize * 1024 * 1024),
```

Comment:
hmm, was this is a bug? `CacheConfig` stores all caches as MB IIRC

Reply:
yes, it seems light client didn't respect the memory budget